frontend/package.json

```diff
@@ -79,6 +80,7 @@
     "react-redux": "5.x",
     "react-router-dom": "4.x",
     "react-router-hash-link": "^1.2.0",
+    "react-select": "1.3",
```

Comment:
We should use the existing dropdown from the console instead of adding a new dependency.